AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -79,53 +80,65 @@ The Alembic exporter can be customized by setting the properties on this compone
 
 | Property:| Function: |
 |:---|:---| 
-|__Output Path__ |Specify the location where the Alembic Exporter will save the ABC file. |
+|__Output Path__ |Specify the location where the Alembic Exporter will save the ABC file. By default the output path is relative to the current Unity project path. |
 |__Archive Type__ |Choose the Alembic format specification, the default is Ogawa and provides smaller and better performance than HDF5.|
 |__Xform Type__ |Choose the transform type. The default is TRS and records the TRS channels for position, rotation, scale of an object. The alternative is matrix (Matrix).|
-|__Time Sampling Frame Rate__ |{TODO}|
-|__Time Sampling Fix Delta Time__ |{TODO}|
+|__Time Sampling Type__ |Choose between Uniform and Acyclic time sampling. In the case of Uniform, the interval between frames on the Alembic side is always constant (1 / Frame Rate seconds). In the case of Acyclic, the delta time on the Unity side is directly the interval between the frames on the Alembic side. The interval is not constant, but the impact on the game progress will be minimal. It is a mode mainly assuming 3D recording of games.|
+|__Time Sampling Frame Rate__ |The frame rate to use for uniform sampling |
+|__Time Sampling Fix Delta Time__ |If enabled, Time.maximumDeltaTime will be set using the frame rate to ensure fixed delta time. In the case of video production this should be the desired behavior, but be careful if you are managing Time.maxDeltaTime yourself. |
 |__Swap Handedness__ |Choose swap handedness to change from a left hand coordinate system (Unity) to a right hand coordinate system (Maya).|
 |__Swap Faces__ |Choose swap faces to reverse the front and back of a face.|
-|__Swap Factor__ |Choose swap factor to convert system units, for example 0.1, converts it to 1/10 size. This also affects position and speed.|
-|__Capture Scope__<br/>"Entire Scene",<br/>"Target Branch" | Choose the scope of the export. By default the entire scene will be exported but it can be configured to export just a branch of the scene. ![Alembic Export Target](images/abc_export_target.png) |
-|__Assume None Skinned Meshes Are Constant__ |{TODO}|
-|__Capture MeshRenderer__ |{TODO}|
-|__Capture SkinnedMeshRenderer__ |{TODO}|
-|__Capture Particle__ |{TODO}|
-|__Capture Camera__ |{TODO}|
-|__Mesh Components Normals__ |{TODO}|
-|__Mesh Components UV1__ |{TODO}|
-|__Mesh Components UV2__ |{TODO}|
-|__Mesh Components Vertex Color__ |{TODO}|
-|__Mesh Components Submeshes__ |{TODO}|
-|__Capture On Start__ |{TODO}|
-|__Ignore First Frame__ |{TODO}|
-|__Max Capture Frame__ |{TODO}|
-|__Detailed Log__ |{TODO}|
-|__Begin Recording__ |{TODO}|
+|__Scale Factor__ |Choose scale factor to convert system units, for example 0.1, converts it to 1/10 size. This also affects position and speed.|
```

Comment:
... Set scale factor to convert between different system units. For example, using 0.1 will convert the Unity units to 1/10 of their value in the resulting Alembic file. This also...